### PR TITLE
[FEATURE] add DKVMN

### PR DESCRIPTION
**Pull request by 0russwest0**
## Description ##
add DKVMN

### What does this implement/fix? Explain your changes.
add DKVMN

#### Pull request type
- [ ] [DATASET] Add a new dataset
- [ ] [BUGFIX] Bugfix
- [x] [FEATURE] New feature (non-breaking change which adds functionality)
- [ ] [BREAKING] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] [STYLE] Code style update (formatting, renaming)
- [ ] [REFACTOR] Refactoring (no functional changes, no api changes)
- [ ] [BUILD] Build related changes
- [ ] [DOC] Documentation content changes
- [ ] [Sync] Synchronization with a repository 
- [ ] [OTHER] Other (please describe): 

#### Changes
add DKVMN

### Does this close any currently open issues?
no

### Any relevant logs, error output, etc?
no

## Checklist ##
Before you submit a pull request, please make sure you have to following:

### Essentials ###
- [x] PR's title starts with a category (e.g. [BUGFIX], [FEATURE], [BREAKING], [DOC], etc)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage and al tests passing
- [x] Code is well-documented (extended the README / documentation, if necessary)
- [x] If this PR is your first one, add your name and github account to [AUTHORS.md](../blob/master/AUTHORS.md)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Review by fannazya — APPROVED** [review]

**Review by fannazya — APPROVED** [review]

**Review by fannazya — CHANGES_REQUESTED** [review]
Please add an example.

**Review by tswsxk — CHANGES_REQUESTED** [review]
The annotations of each function are expected to be strictly follow numpy-style.

**Comment by tswsxk** [comment]
@fannazya why I do not see any feedback from you since last month?

File: examples/DKVMN/DKVMN.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# coding: utf-8
+# 2022/3/18 @ ouyangjie
+
+
+from EduKTM import DKVMN
+from load_data import DATA
```
Comment (tswsxk) [inline examples/DKVMN/DKVMN.py:6]: DATA is a constant-style name, use Data if needed

File: examples/DKVMN/load_data.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                if len(Q) > self.seqlen:
+                    n_split = math.floor(len(Q) / self.seqlen)
+                    if len(Q) % self.seqlen:
+                        n_split = n_split + 1
+                #print('n_split:',n_split)
+                for k in range(n_split):
+                    question_sequence = []
+                    answer_sequence = []
+                    if k == n_split - 1:
+                        endINdex  = len(A)
```
Comment (tswsxk) [inline examples/DKVMN/load_data.py:47]: variable name should be UnderScoreCase rather than CamelCase

**Review by tswsxk — CHANGES_REQUESTED** [review]

**Review by tswsxk — APPROVED** [review]

**Review by tswsxk — APPROVED** [review]

**Pull request merged**